src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/increment-session-attribute-count.mdx

```diff
@@ -0,0 +1,710 @@
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/increment-attribute
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/incrementattribute-ios-sdk-api
+  - /docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/increment-attribute
+---
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="android">
+            Android
+        </TabsBarItem>
+        <TabsBarItem id="ios">
+            iOS
+        </TabsBarItem>
+        <TabsBarItem id="capacitor">
+            Capacitor
+        </TabsBarItem>
+        <TabsBarItem id="cordova">
+            Cordova
+        </TabsBarItem>
+        <TabsBarItem id="flutter">
+            Flutter
+        </TabsBarItem>
+        <TabsBarItem id="react">
+            React Native
+        </TabsBarItem>
+        <TabsBarItem id="xamarin">
+            Xamarin
+        </TabsBarItem>
+        <TabsBarItem id="maui">
+            .NET MAUI
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="android">
+## Syntax [#syntax]
+
+### Java [#java]
+
+```java
+NewRelic.incrementAttribute(string $name[, float $value])
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+NewRelic.incrementAttribute(name: String!, value : Double)
+```
+
+## Description [#description]
+
+Increments the count of an attribute. Overwrites its previous value and type each time it is called.
+
+When passed with only a name value, this method increments the count for the specified attribute by 1. If the attribute does not exist, it creates the attribute with a value of 1. When passed an optional float value, it increments the count for the specified attribute by the float value.
+
+For context on how to use this API, see [Send custom attributes and events](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/work-android-sdk-api#attributes-events-insights).
+
+## Parameters [#parameters]
+
+<table>
+  <thead>
+    <tr>
+      <th width="30%">
+        Parameter
+      </th>
+
+      <th width={200}>
+        Type
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$name`
+      </td>
+
+      <td>
+        `string`
+      </td> 
+
+      <td>
+       Required. The name of the attribute.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        `$value`
+      </td>
+
+      <td>
+        `float`
+      </td>
+      <td>
+        Optional. The attribute is incremented by this float value.
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+## Return values [#return-values]
+
+Returns `true` if recorded successfully, or `false` if not.
+
+## Examples [#examples]
+
+Here's an example to increment the count for the specified attribute by 1. If the attribute does not exist, it creates the attribute with a value of 1:
+
+### Java [#java]
+
+```java
+boolean incremented = NewRelic.incrementAttribute("rate");
+```
+
+### Kotlin [#kotlin]
+
+```kotlin
+
+```
```

Comment:
@ndesai-newrelic Can you please provide a kotlin example equivalent to the Java example?

Reply:
1)         val incremented = NewRelic.incrementAttribute("rate")
2)           val incremented: Boolean = NewRelic.incrementAttribute("rate", 9999.99)

Reply:
```suggestion
```kotlin
    val incremented = NewRelic.incrementAttribute("rate")

     val incremented: Boolean = NewRelic.incrementAttribute("rate", 9999.99)

```
```